scripts/start-services-for-e2e-tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -euo pipefail
+
+readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+readonly UI_NAME=make-recall-decision-ui
+readonly API_NAME=make-recall-decision-api
+readonly UI_DIR="${SCRIPT_DIR}/../../${UI_NAME}"
+readonly API_DIR="${SCRIPT_DIR}/../../${API_NAME}"
+
+pushd "${API_DIR}"
+printf "\n\nBuilding/starting API components...\n\n"
+docker-compose pull
```

Comment:
in the UI readme the order is build -> pull not pull -> build
not sure if it matters tho?

Reply:
Makes no difference